test/transcoding/SPV_INTEL_token_type/token_type_intel_negative.ll

```diff
@@ -0,0 +1,37 @@
+;; Check whether the translator reports an error for a module with token type
+;; if SPV_INTEL_token_type extension is not used.
```

Comment:
Is second semicolon required?
```suggestion
; Check whether the translator reports an error for a module with token type
; if SPV_INTEL_token_type extension is not used.
```

Reply:
Nope, it's just my habit (may be bad) to but double semicolon to add a comment in LIT. Applied.